README.md

```diff
@@ -333,6 +334,7 @@ Note that it only runs on CI.
 | Key & Env Var | Description | Default Value
 |-----------------|--------------------|---|
 | `api_token` <br/> `AC_API_TOKEN` | The API Token to interact with AppCenter APIs | |
+| `owner_name` <br/> `AC_OWNER_NAME` | The owner/organization name in AppCenter | `Fueled` | |
```

Comment:
Looks like you have an extra ` |` at the end. The default value is the last column. I must admit, markdown is not the best tool to create charts 😅

Reply:
🙈

Reply:
Thanks for flagging! I thought a double pipe meant an end of row 🙈 
I guess it's time for me to learn some Markdown.